code/game/machinery/bioprinter.dm

```diff
@@ -110,21 +110,18 @@
 	name = "prosthetic organ fabricator"
 	desc = "It's a machine that prints prosthetic organs."
 	icon_state = "roboprinter"
+	print_delay = 300 //~30 second print time default. 18 seconds with pico manipulators
 
 	products = list(
-		BP_HEART    = list(/obj/item/organ/internal/heart,      25),
-		BP_LUNGS    = list(/obj/item/organ/internal/lungs,      25),
-		BP_KIDNEYS  = list(/obj/item/organ/internal/kidneys,    20),
-		BP_EYES     = list(/obj/item/organ/internal/eyes,       20),
-		BP_LIVER    = list(/obj/item/organ/internal/liver,      25),
-		BP_L_ARM    = list(/obj/item/organ/external/arm,        65),
-		BP_R_ARM    = list(/obj/item/organ/external/arm/right,  65),
-		BP_L_LEG    = list(/obj/item/organ/external/leg,        65),
-		BP_R_LEG    = list(/obj/item/organ/external/leg/right,  65),
-		BP_L_FOOT   = list(/obj/item/organ/external/foot,       40),
-		BP_R_FOOT   = list(/obj/item/organ/external/foot/right, 40),
-		BP_L_HAND   = list(/obj/item/organ/external/hand,       40),
-		BP_R_HAND   = list(/obj/item/organ/external/hand/right, 40)
+		BP_HEART    = list(/obj/item/organ/internal/heart,      45),
+		BP_LUNGS    = list(/obj/item/organ/internal/lungs,      45),
+		BP_KIDNEYS  = list(/obj/item/organ/internal/kidneys,    40),
+		BP_EYES     = list(/obj/item/organ/internal/eyes,       40),
+		BP_LIVER    = list(/obj/item/organ/internal/liver,      45),
+		BP_L_ARM    = list(/obj/item/robot_parts/l_arm,         100),//Old ones didn't work, plus these ones can be used in medibot crafting
+		BP_R_ARM    = list(/obj/item/robot_parts/r_arm,         100),
+		BP_L_LEG    = list(/obj/item/robot_parts/l_leg,         100),
+		BP_R_LEG    = list(/obj/item/robot_parts/r_leg,         100),
```

Comment:
What do you mean "they didn't work"?

Reply:
They were just a hand or an arm (with no hand I think). I couldn't use my hand, it was being a bit weird. I forget if there was something with 2 arms showing up. Also it makes it easy to make medibots.

Reply:
There is a `full` subtype that should be used. It has both arm and a hand. I'll change medibot to require a robotized arm instead of robo part

Reply:
kk